src/redux/features/observations/observationsThunks.ts

```diff
@@ -38,10 +39,16 @@ export const requestObservationsResults = (organizationId: number) => {
 /**
  * Fetch planting site observation results
  */
-export const requestPlantingSiteObservationsResults = (organizationId: number, plantingSiteId: number) => {
+export const requestPlantingSiteObservationsResults = (
+  organizationId: number,
+  plantingSiteId: number,
+  adHoc?: boolean
+) => {
```

Comment:
I assume it simplifies the code to add an `adHoc` param rather than creating a separate set of functions for ad hoc observations? I could see an argument being made to create separate functions, but this change seems reasonable to me 👍

Reply:
Yes, the responses are the same, that's why I used same functions for this, so we don't have a lot of repeated code